specs/fault-proof/stage-one/optimism-portal.md

```diff
@@ -229,13 +272,14 @@ Allows a user to [prove](#proven-withdrawal) a withdrawal transaction.
 
 Checks that a withdrawal transaction can be [finalized](#finalized-withdrawal).
 
+- MUST revert if the withdrawal being finalized has already been finalized.
 - MUST revert if the withdrawal being finalized has not been proven.
+- MUST revert if the withdrawal was proven at a timestamp less than the creation timestamp of the
```

Comment:
Sounds impossible, but ok to keep in the implementation as a sanity check imo.

Reply:
Yeah I think it's impossible but no harm in keeping the check.